qcodes_contrib_drivers/drivers/Vaunix/LDA.py

```diff
@@ -51,8 +52,8 @@ def __init__(self, name: str,
                 (serial:55102). Does not communicate with physical devices. For
                 testing purposes.
         """
+        begin_time = time.time()
```

Comment:
Why do we need this?

Reply:
Otherwise the `connect_message` displays 0.00 seconds, it's mostly just cosmetic

Reply:
Thanks, fair enough. I believe you already tested this improved version with a real instrument?

Reply:
Yes, that's how we found these bugs

Reply:
Thanks.